test/test_simulation_results.jl

```diff
@@ -433,7 +432,9 @@ function test_simulation_results(file_path::String, export_path; in_memory = fal
 end
 
 @testset "Test simulation results" begin
-    for in_memory in (true, false)
+    # TODO: hdf5 store is not currently working
```

Comment:
Does this work now?

Reply:
yeah, it works. I should remove this comment